docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,736 @@
+  "info": {
+    "name": "user",
+    "description": ""
+  },
+  "attributes": {
+    "username": {
+      "type": "string",
+      "minLength": 3,
+      "unique": true,
+      "configurable": false,
+      "required": true
+    },
+    "email": {
+      "type": "email",
+      "minLength": 6,
+      "configurable": false,
+      "required": true
+    },
+    "provider": {
+      "type": "string",
+      "configurable": false
+    },
+    "password": {
+      "type": "password",
+      "minLength": 6,
+      "configurable": false,
+      "private": true
+    },
+    "resetPasswordToken": {
+      "type": "string",
+      "configurable": false,
+      "private": true
+    },
+    "confirmed": {
+      "type": "boolean",
+      "default": false,
+      "configurable": false
+    },
+    "blocked": {
+      "type": "boolean",
+      "default": false,
+      "configurable": false
+    },
+    "role": {
+      "model": "role",
+      "via": "users",
+      "plugin": "users-permissions",
+      "configurable": false
+    },
+    "products": {
+      "collection": "product",
+      "via": "owner"
+    }
+  }
+}
+```
+
+**After**
+
+`./extensions/users-permissions/models/User.settings.json`
+
+```json
+{
+  "connection": "default",
+  "collectionName": "users-permissions_user",
+  "info": {
+    "name": "user",
+    "description": ""
+  },
+  "attributes": {
+    "username": {
+      "type": "string",
+      "minLength": 3,
+      "unique": true,
+      "configurable": false,
+      "required": true
+    },
+    "email": {
+      "type": "email",
+      "minLength": 6,
+      "configurable": false,
+      "required": true
+    },
+    "provider": {
+      "type": "string",
+      "configurable": false
+    },
+    "password": {
+      "type": "password",
+      "minLength": 6,
+      "configurable": false,
+      "private": true
+    },
+    "resetPasswordToken": {
+      "type": "string",
+      "configurable": false,
+      "private": true
+    },
+    "confirmed": {
+      "type": "boolean",
+      "default": false,
+      "configurable": false
+    },
+    "blocked": {
+      "type": "boolean",
+      "default": false,
+      "configurable": false
+    },
+    "role": {
+      "model": "role",
+      "via": "users",
+      "plugin": "users-permissions",
+      "configurable": false
+    },
+    "products": {
+      "collection": "product",
+      "via": "owner"
+    }
+  }
+}
+```
+
+#### Admin
+
+For now you won't be able to customize a plugin's admin. This feature will come soon so keep an eye out for this feature.
+
+### Migrating local plugins
+
+If you have local plugins (plugins in the `./plugins` that don't exist on npm) you can leave them in the plugins folder.
+
+The only difference is that the admin of a local plugin is ignored for the moment.
+
+## Migrating `api`
+
+### Migrating controllers and services
+
+In the `beta` we are introducing the `Core API`, which is replacing the templates that were generated before.
```

Comment:
In the `beta`, we are introducing the `Core API`, which is replacing the templates which were generated before.